css/css-fonts/variations/font-style-parsing.html

```diff
@@ -33,7 +33,10 @@
             { style: "oblique 20deg a",   expectedResult: false, message: "'oblique' and angle followed by non-number is invalid" },
             { style: "oblique -",         expectedResult: false, message: "'oblique' followed by isolated minus is invalid" },
             { style: "oblique - 20deg",   expectedResult: false, message: "'oblique' followed by minus and angle separated by space is invalid" },
-            { style: "oblique -a",        expectedResult: false, message: "'oblique' followed by minus and non-number is invalid" }
+            { style: "oblique -a",        expectedResult: false, message: "'oblique' followed by minus and non-number is invalid" },
+            { style: "oblique calc(50deg)",     expectedResult: true,  message: "'oblique' followed by calc is valid" },
+            { style: "oblique calc(-120deg)",   expectedResult: true,  message: "'oblique' followed by calc is valid even if it must be clamped (no computation)" },
```

Comment:
Can you add cases for different angle units? E.g. calc(2 * 0.1rad)

Reply:
I doubt that this is relevant. A browser that would fail these tests but not the existing ones would have an issue with css-value-and-units not font-style parsing.

Reply:
Are there tests for css-values to cover calc() for angles? I don't see any, but I may be missing it. 

There are specific tests covering other properties and units. We should have tests for angles, either here or in css-values folder